src/commands/createApicEnv.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
+import { IActionContext, UserCancelledError } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { ApiCenterService } from "../azure/ApiCenter/ApiCenterService";
+import { ApiCenterEnvironment } from "../azure/ApiCenter/contracts";
+import { ApiCenterEnvironmentServerType, ContinueToSkip, EnvironmentKind } from "../constants";
+import { ext } from "../extensionVariables";
+import { ApiCenterTreeItem } from "../tree/ApiCenterTreeItem";
+import { EnvironmentsTreeItem } from "../tree/EnvironmentsTreeItem";
+import { UiStrings } from "../uiStrings";
+import { GeneralUtils } from "../utils/generalUtils";
+export async function generateApicEnv(context: IActionContext, node?: EnvironmentsTreeItem): Promise<void> {
+    if (!node) {
+        const apiCenterNode = await ext.treeDataProvider.showTreeItemPicker<ApiCenterTreeItem>(ApiCenterTreeItem.contextValue, context);
+        node = apiCenterNode.environmentsTreeItem;
+    }
+
+    if (!node) {
+        return;
+    }
+
+    const apiEnvName = await vscode.window.showInputBox({
+        title: UiStrings.EnterApiCenterEnvironmentName, prompt: UiStrings.InputValidEnvironmentName, ignoreFocusOut: true, validateInput: GeneralUtils.validateInputForTitle
+    });
+
+    if (!apiEnvName) {
+        throw new UserCancelledError();
+    }
+
+    const envKind = await vscode.window.showQuickPick(Object.values(EnvironmentKind), { placeHolder: UiStrings.SelectEnvironmentKind });
+    if (!envKind) {
+        throw new UserCancelledError();
+    }
+
+    const serverType = await vscode.window.showQuickPick([ContinueToSkip, ...Object.values(ApiCenterEnvironmentServerType)], { placeHolder: UiStrings.SelectApicEnvironmentServerType });
```

Comment:
Better to add vscode.QuickPickItemKind.Separator
Refer to https://github.com/microsoft/vscode-azureapicenter/blob/dev/src/commands/generateHttpFile.ts#L95C17-L101C20